firefox-desktop/views/firefox_desktop_usage_2021.view.lkml

```diff
@@ -0,0 +1,166 @@
+view: firefox_desktop_usage_2021 {
+  sql_table_name: `mozdata.telemetry.firefox_desktop_usage_2021`
+    ;;
+
+  dimension: activity_segment {
+    type: string
+    sql: ${TABLE}.activity_segment ;;
+  }
+
+  dimension: campaign {
+    type: string
+    sql: ${TABLE}.campaign ;;
+  }
+
+  measure: cdou_computed {
```

Comment:
Any particular reason to have this? I find them odd because the sorting has to be correct, and by default it's wrong (it sorts DESC).

Reply:
Yea this is now a bit out of date, I was computing this as a temporary fix while Jeff worked on the PR to fix the underlying `cdou` column. I'll switch this back now that this fix has landed.